### PR TITLE
ResNet-50 || ImageNet Inference || Perseus AI Cloud Acceleration team in Alibaba Cloud

### PR DESCRIPTION
**Pull request by ShaohuaWu2018**
Alibaba Cloud submissions for ImageNet inference with resnet50 on ecs.gn5i-c8g1.2xlarge

### CODE REVIEW

File: ImageNet/inference/AlibabaCloud_resnet50_1p4_ecs_tensorflow.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+    "version": "v1.0",
+    "author": "Perseus AI Cloud Acceleration team in Alibaba Cloud",
+    "authorEmail": "wsh164800@alibaba-inc.com, youliang.yl@alibaba-inc.com",
+    "framework": "TensorFlow",
```
Comment (codyaustun) [inline ImageNet/inference/AlibabaCloud_resnet50_1p4_ecs_tensorflow.json]: Please include TensorFlow version number.

File: ImageNet/inference/AlibabaCloud_resnet50_1p4_ecs_tensorflow.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "codeURL": "https://github.com/ali-perseus/DAWNBench_Inference",
```
Comment (codyaustun) [inline ImageNet/inference/AlibabaCloud_resnet50_1p4_ecs_tensorflow.json:6]: Can you add the commit hash either to the `codeURL` or as a separate `commitHash` field?

**Review by codyaustun — CHANGES_REQUESTED** [review]
Looks good overall. Just a few minor changes.

File: ImageNet/inference/AlibabaCloud_resnet50_1p4_ecs_tensorflow.json (hunk 1 above)
Comment (ShaohuaWu2018) [reply ImageNet/inference/AlibabaCloud_resnet50_1p4_ecs_tensorflow.json]: Thanks for your kindness and great efforts on this PR. The version number will be added.

File: ImageNet/inference/AlibabaCloud_resnet50_1p4_ecs_tensorflow.json (hunk 2 above)
Comment (ShaohuaWu2018) [reply ImageNet/inference/AlibabaCloud_resnet50_1p4_ecs_tensorflow.json:6]: Thanks, I will add a separate commitHash field to include commit hash.

**Comment by ShaohuaWu2018** [comment]
The tensorflow version number and commit hash for DAWNBench_Inference code have been added. Thank you for your time and looking forward to your response.

**Review by codyaustun — APPROVED** [review]

**Comment by codyaustun** [comment]
Thanks! Looks good to me.

**Comment by codyaustun** [comment]
Actually, I have one more question. Can you point me to pricing information for Alibaba Cloud? I just want to double check that the cost field is correct.

**Comment by ShaohuaWu2018** [comment]
Thanks! I am pleased to make an introduction to show how to get the price information from web site of Alibaba Cloud.

**Comment by ShaohuaWu2018** [comment]
Here is the link to show the price for Alibaba Cloud instances: 
https://cn.aliyun.com/price/product?spm=5176.8789780.1092586.1.2d0257a8rKsPw1#/ecs/detail

On this website, please scroll down and find the instance with the name of "ecs.gn5i-c8g1.2xlarge".
You will see there are 9 columns, and the price of on-demand is shown in the fourth column.

I am so sorry that the price page is in Chinese. I have made a translation for the titles of all these 9 columns as below, 
Column 1: vCPU
Column 2: Type of instances.
Column 3: Memory.
Column 4: On-demand price (per hour).
Column 5: Monthly price.
Column 6: Monthly reduced price.
Column 7: Yearly price.
Column 8: Monthly price for 3 years.
Column 9: Monthly price for 5 years.

Please note: the price listed is in Chinese Yuan (CNY), please convert it to US dollar by an exchange rate of 0.15 that is the rate we checked on 2018-12-12, the day of this submission. 

For any questions, please do not hesitate to contact me. Thanks.

**Comment by codyaustun** [comment]
Great! That looks correct.  ((1000 * 3600) / 4.218 ) * 1.54e-06 / 0.15 is roughly 8.75 CNY. Thanks for the link and translation!

**Pull request merged**